src/pages/api/v1/sign-in.ts

```diff
@@ -36,6 +36,8 @@ async function authenticateUser(
         publicKey: user.publicKey,
         algorithm: typeOfAlgorithm,
       };
+      // eslint-disable-next-line no-console
```

Comment:
- require deprecated annotation for console.log
```
// Deprecated: temporary console.log debug information (20240819 - Anna)
```

Reply:
- done